### PR TITLE
add authenticator param to getAwsCredentials

### PR DESCRIPTION
**Pull request by dagovalsusa**
add authenticator param to getAwsCredentials function
to authenticate with third authentication method (as Facebook, Google)

### CODE REVIEW

**Comment by furaiev** [comment]
@dagovalsusa thank you for your PR, could you please add short description to Readme with an example of using third authentication method ?

**Comment by dagovalsusa** [comment]
Of course, 
ASAP

File: lib/src/cognito_credentials.dart [hunk 1]
```diff
@@ -63,6 +63,8 @@ class CognitoCredentials {
       secretAccessKey = data['Credentials']['SecretKey'];
       sessionToken = data['Credentials']['SessionToken'];
 
+      print(accessKeyId);
```
Comment (JohnyBalian) [inline lib/src/cognito_credentials.dart]: no need to print

Comment (dagovalsusa) [reply lib/src/cognito_credentials.dart]: ok

**Comment by furaiev** [comment]
Thank you for contribution @dagovalsusa @JohnyBalian !
Available as of 0.1.7

**Pull request merged**